mesh/mesh.go

```diff
@@ -30,6 +30,10 @@ type StateUpdater interface {
 	ApplyRewards(layer LayerID, miners map[string]struct{}, underQuota map[string]struct{}, bonusReward, diminishedReward *big.Int)
 }
 
+type AtxDb interface {
```

Comment:
we don't use this interface, if its not for future implementation
we can delete this